### ISSUE TITLE
Update peerDependencies

### ISSUE THREAD

**Issue by masiamj** [seiyria/ng2-pnotify#1]
Trying to install; however, peerDependencies are unsatisfied with latest Angular 2 release candidate.

**Comment by seiyria** [comment]
That's true. I haven't upgraded anything to the latest angular. Feel free to toss me a PR if you need it.

**Comment by masiamj** [comment]
Updated dep. Tried to push to new branch, but denied. Maybe add permission? Great port of project btw

**Comment by seiyria** [comment]
You need to make a pull request from your fork.

**Comment by niklas-dahl** [comment]
any updates on this?
would like to use this with latest angular

**Comment by seiyria** [comment]
No. Submit a pull request if you want it fixed.
